### ISSUE TITLE
About the reproduced results

### ISSUE THREAD

**Issue by NieCha** [Allencheng97/Multilinear_Operator_Networks#1]
Can you share the training logs on Imagenet regarding the MONet_T model, I can't get 77.0 top-1 ACC in my replicated results. This could be a problem with inconsistent startup scripts.

**Comment by Allencheng97** [comment]
Hi NieCha,

Since this work was done a long time ago and I am no longer working at EPFL, some files have been lost. However, fortunately, I found a backup of the MONet_T training logs on Wandb, which is split into two files corresponding to the purple and pink curves shown in the image, respectively.
![image](https://github.com/user-attachments/assets/32a0dfe8-a585-46dd-bb26-ac31ab44c0c9)

[output.log](https://github.com/user-attachments/files/18238047/output.log)
[output (1).log](https://github.com/user-attachments/files/18238045/output.1.log)

One trick to get higher performance is not enable ema too early. For example, in the shown case I enable ema at 135 epoches.

**Comment by Allencheng97** [comment]
Here's a detail hyperparameter setting json file from wandb I hope could help you get better performance
[wandb-metadata.json](https://github.com/user-attachments/files/18238171/wandb-metadata.json)

**Comment by NieCha** [comment]
Thanks, that's very helpful!^o^

**Comment by NieCha** [comment]
Hi. Unfortunately, I'm still not getting 77.0% accuracy on imagenet1k. I found that the first 10 epochs already achieved a lower precision than the results in your log (almost 2.5% lower). Could you please further provide the startup script~

**Comment by Allencheng97** [comment]
Hi, The shown result are running on 80GB VRAM A100. I remember with amp training the batch size is 448. based on my experience if your are using the default layernorm version. The impact of the batchsize on the results is relatively small.  What's the final result you get? Is it diverage a lot from the training log I share?

**Comment by NieCha** [comment]
Yeah, it's about three or four points lower. Monet is a great work, so I'm trying to keep doing it.

**Comment by Allencheng97** [comment]
> Yeah, it's about three or four points lower. Monet is a great work, so I'm trying to keep doing it.

Quite strange, Do you enable ema in the later epoch? After ensembing you should get a 2-3 points improved after a few epoches.

**Comment by NieCha** [comment]
Exactly~ And I notice the training accuracy per epoch always lower than yours in provided log, 70.5 vs. 73.758 at 135 epoch

**Comment by Allencheng97** [comment]
> Exactly~ And I notice the training accuracy per epoch always lower than yours in provided log, 70.5 vs. 73.758 at 135 epoch

I am not sure why this happened.  Since which epoch it is lower? I feel it may be caused to the total batch size of distributed training. I use 4 80GB A100 for training, each GPU is 256.

**Comment by NieCha** [comment]
I will use gradient accumulation to enlarge global batch size to 256 * 4 and re-experiment.  Also is there a chance to add your wechat for better communication~

**Comment by Allencheng97** [comment]
Sure, please leave your wechat number here.

**Comment by NieCha** [comment]
ClareNie
